packages/app/src/app/pages/Curator/index.tsx

```diff
@@ -65,7 +76,8 @@ const Curator: React.FC = () => {
         </SubTitle>
 
         <Buttons>
-          Most popular sandboxes in the:
+          <span>Most popular sandboxes in the:</span>
```

Comment:
```suggestion
          Most popular sandboxes in the:
```

Not sure why you added the `span`? This causes a visual bug, missing margin. So needs to be reverted. Other than that, looks great 😄